Docker.md

```diff
@@ -159,6 +160,34 @@ $ docker logs -f [name-of-container]
 
 which will continuously stream the logs to the terminal. To exit, type ctrl+c. To just print the logs and exit, run the same command without the -f flag.
 
+> **NOTE**:
+> To login, use the username `admin` and password `password`. These are the default values used to create a new user.
+
+## Providing User Credentials
+
+User credentials can be passed via the `-e` option to Archivy as follows
+
+```sh
+$ docker run -d --name archivy-test -p 5000:5000 -e ADMIN_USER="user" -e ADMIN_PASSWORD="pass" uzayg/archivy
+```
+
+If value for `ADMIN_USER` and `ADMIN_PASSWORD` are not provided, the default values of `admin` and `password` are used. You can change the login password after logging in, if necessary.
+
+| OPTION           | DEFAULT    |
+| ---------------- | ---------- |
+| `ADMIN_USER`     | `admin`    |
+| `ADMIN_PASSWORD` | `password` |
+
+## Resuming Stopped Instance
+
+If you have an instance of Archivy running that you wish to restart with previously saved data(like users and corresponding passwords), you __must__ provide the `run` command to the `docker` command as follows:
+
+```sh
+$ docker run -d --name archivy-test -p 5000:5000 -v /path/to/data/dir:/archivy uzayg/archivy run
+```
+
+This will start the container will the `/path/to/data/dir` bind-mounted to the `/archivy` directory.
+
 ## Running container in interactive mode
 
 You can also pass commands to the container by appending it to the docker run command. Keep in mind that the container will execute your commands and not run Archivy. This is useful if you want to find out what is going on inside the container. For example,
```

Comment:
Does this mean users could directly call `create-admin` for example? In that case, could they just do that instead of the env variables?

Or are there drawbacks to this approach?